build.gradle

```diff
@@ -34,6 +34,6 @@ allprojects {
     }
 }
 
-task clean(type: Delete) {
+task delete(type: Delete) {
```

Comment:
Why?

Reply:
The spotless plugin had a predefined task named clean which conflict with this one, so I changed the name of the task.

Reply:
@iamareebjamal Please review